versioned/tiered/rocksdb/src/main/java/com/dremio/nessie/versioned/store/rocksdb/RocksDBStore.java

```diff
@@ -106,8 +106,15 @@ public void start() {
         final List<ColumnFamilyHandle> columnFamilyHandles = new ArrayList<>();
         transactionDB = OptimisticTransactionDB.open(dbOptions, dbPath, columnFamilies, columnFamilyHandles);
         rocksDB = transactionDB.getBaseDB();
-        valueTypeToColumnFamily = IntStream.rangeClosed(1, ValueType.values().size()).boxed().collect(
-          ImmutableMap.toImmutableMap(k -> ValueType.values().get(k - 1), columnFamilyHandles::get));
+        final ImmutableMap.Builder<ValueType<?>, ColumnFamilyHandle> builder = new ImmutableMap.Builder<>();
+        for (ColumnFamilyHandle handle : columnFamilyHandles) {
+          final String valueTypeName = new String(handle.getName(), UTF_8);
+          if (!valueTypeName.equals(DEFAULT_COLUMN_FAMILY)) {
+            // TODO: byValueName returns the valueName. We need to compare the defaultTableSuffix.
```

Comment:
I'm not sure what this means - does this not work as is? byValueName returns a ValueType associated with the actual string name.

Reply:
Correct, however handle.getName() has been set to the ValueType.defaultTableSuffix not ValueType.valueName. There is no method to find the ValueType from defaultTableSuffix.

Reply:
I have added a filter in the stream to find ValueType using getTableName() which seems to work.

Reply:
Reverted to previous version and now name the ColumnFamily with ValueName.